### PR TITLE
KAFKA-5215. Small JavaDoc fix for AdminClient#describeTopics

### PR DESCRIPTION
**Pull request by cmccabe**

### CODE REVIEW

File: clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java [hunk 1]
```diff
@@ -131,7 +131,7 @@ public DescribeTopicsResults describeTopics(Collection<String> topicNames) {
     }
 
     /**
-     * Descripe an individual topic in the cluster.
+     * Describe some topics in the cluster.
      *
      * Note that if auto.create.topics.enable is true on the brokers,
      * AdminClient#describeTopic(topicName) may create a topic named topicName.
```
Comment (theduderog) [inline clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java]: Fix this reference?  There is no `AdminClient#describeTopic(topicName)` method

File: clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java [hunk 2]
```diff
@@ -131,10 +131,10 @@ public DescribeTopicsResults describeTopics(Collection<String> topicNames) {
     }
 
     /**
-     * Descripe an individual topic in the cluster.
+     * Describe some topics in the cluster.
      *
      * Note that if auto.create.topics.enable is true on the brokers,
-     * AdminClient#describeTopic(topicName) may create a topic named topicName.
+     * describeTopics(topicName, ...) may create a topic named topicName.
```
Comment (theduderog) [inline clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java:137]: Nit: `describeTopics(Collection<String> topicNames, ...)`

Comment (cmccabe) [reply clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java:137]: Hmm.  This is intended to point out that 'topicName' is part of the collection of topics being described.  It's not a reference to a different function.

**Review by ijuma — APPROVED** [review]
Thanks for the PR, LGTM.

**Pull request closed without merging**